js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -464,3 +480,9 @@ function removeFromMessageQueue(queueObj) {
     mabMessageQueue.splice(messageObjIndex, 1);
   }
 }
+
+function trimBannerText(message: string = undefined, maxLength = 128) {
```

Comment:
@Climbatize @munir-rakuten Does the iOS/Android SDK support receiving `undefined`? We need to make sure this behaves as expected on the native side - i.e. The `bannerMessage` property should be optional in the native model.